documentation/src/main/asciidoc/server_guide/server_guide.adoc

```diff
@@ -607,6 +607,45 @@ The REST connector differs from the above connectors because it piggybacks on th
 
 ----
 
+==== Multi-tenant router
+The Multi-tenant router endpoint facades one or more REST or Hot Rod connectors. Its main purpose is to forward client requests into proper container.
+
+In order to properly configure the routing, `socket-binding` attributes of other connectors must be disabled and additional attribute `name` must be used as shown below:
+
+[source,xml]
+----
+
+<rest-connector name="rest-1" cache-container="local"/>
+<rest-connector name="rest-2" cache-container="local"/>
+<hotrod-connector name="hotrod-1" cache-container="local" />
+<hotrod-connector name="hotrod-1" cache-container="local" />
+
+----
+
+The next step is to add a new `router-connector` endpoint and configure how other containers will be accessed. Note that Hot Rod connectors require using TLS/SNI and REST connectors require using prefix in the URL:
+
+[source,xml]
+----
+
+<router-connector hotrod-socket-binding="hotrod" rest-socket-binding="rest" keep-alive="true" tcp-nodelay="false" receive-buffer-size="1024" send-buffer-size="1024">
+    <hotrod name="hotrod-1" >
+        <sni host-name="hotrod-1" security-realm="SSLRealm1"/>
+    </hotrod>
+    <hotrod name="hotrod-2" >
+        <sni host-name="hotrod-2" security-realm="SSLRealm2"/>
+    </hotrod>
+    <rest name="rest-1">
+        <prefix path="rest-1" />
+    </rest>
+    <rest name="rest-2">
+        <prefix path="rest-2" />
+    </rest>
+</router-connector>
+
+----
+
+With the following configuration, Hot Rod clients will access `rest-1` connector when using SNI Host Name "hotrod-1". REST clients will need to use the following context to access "rest-1" connector - `http://<server_ip>:<server_port>/rest/rest-1`.
```

Comment:
s/context/URL/?

Also, try to start each sentence on a new line, it will make things easier for the reviewers of the next pull request touching this.